### PR TITLE
Add API to prepare for adding semantics editor

### PR DESCRIPTION
**Pull request by lilyx**

### CODE REVIEW

File: src/Semantics.js [hunk 1]
```diff
@@ -451,6 +459,14 @@ Semantics.createSemantics = function(grammar, optSuperSemantics) {
     s.extendOperationOrAttribute.call(s, 'attribute', name, actionDict);
     return proxy;
   };
+  proxy.getActionDict = function(operationOrAttributeName) {
```
Comment (alexwarth) [inline src/Semantics.js]: This is not part of the "public" API, so let's prepend the name with a `_`.

File: src/Semantics.js [hunk 2]
```diff
@@ -451,6 +459,14 @@ Semantics.createSemantics = function(grammar, optSuperSemantics) {
...
+    var action = s.operations[operationOrAttributeName] || s.attributes[operationOrAttributeName];
+    if (!action) {
+      throw new Error('There is no action has name ' + operationOrAttributeName +
```
Comment (alexwarth) [inline src/Semantics.js]: How about `operationOrAttributeName + ' is not a valid operation or attribute name in this semantics'`?

**Comment by alexwarth** [comment]
I just added a couple of small comments. This looks great, it's just missing a unit test.

**Comment by lilyx** [comment]
Updated with unit test.

**Pull request merged**